### PR TITLE
Translation to Spanish: Update Gear Up for Success and Basic Ziplines

### LINKED ISSUE #6923: Updating Spanish translation on new challenges
- [ ] Basic Front End Development Projects
- [ ] new Gear Up For Success challengesWe're introducing some new challenges

### PR DESCRIPTION
**Pull request by cuent**
Translate to Spanish new challenges of Update Gear Up for Success and  Basic Ziplines files. 

Closes #6923

### CODE REVIEW

**Comment by BerkeleyFalse** [comment]
@cuent thanks for the PR.
Do not include issue numbers and following [keywords](https://help.github.com/articles/closing-issues-via-commit-messages/#keywords-for-closing-issues) in commit messages.
Please, review our [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md), thank you!.

**Comment by bugron** [comment]
@cuent, please do not ignore @BerkeleyFalse's comments and remember, that your PRs should follow our [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md). Provide more descriptive commit message next time. But if you want to fix this now you can `git commit --amend` your commit, edit commit message and then force push to your branch: `git push -f YOUR_ORIGIN_HERE fix/gear-up-for-success-and-basic-ziplines`.

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 1]
```diff
@@ -76,25 +76,25 @@
...
+          "Un gif mostrando como pulsar el botón \"follow\" para seguir las publicaciones de Free Code Camp.",
+          "Una vez que inicies sesión puedes ir al canal de publicaciones Medium de Free Code Camp y pulsar en \"follow\". Nuestros campistas publican varios artículos cada semana.",
+          "https://medium.freecodecamp.com"
         ]
       ]
     },
     {
       "id": "560ade65cb82ac38a17513c2",
-      "title": "Watch us Code Live on Twitch.tv",
+      "title": "Miranos programar en vivo por Twitch.tv",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: `Míranos` en lugar de `Miranos`

File: seed/challenges/01-front-end-development-certification/basic-ziplines.json [hunk 2]
```diff
@@ -167,18 +167,18 @@
...
+        "<strong>Regla #1:</strong> No veas el código del proyecto de ejemplo en CodePen. Encuentra la forma de hacerlo por tu cuenta.",
+        "<strong>Regla #2:</strong> Satisface las siguientes <a href='http://en.wikipedia.org/wiki/User_story' target='_blank'>historias de usuario</a>. Usa cualquier librería que necesites. Dale tu estilo personal.",
+        "<strong>Historia de usuario:</strong> Puedo acceder a todo el contenido de la página del portafolio con sólo desplazarme en la ventana.",
+        "<strong>Historia de usuario:</strong> Puedo pulsar diferentes botones que me llevarán a las páginas de las diferentes cuentas de redes sociales del creador del portafolio.",
+        "<strong>Historia de usuario:</strong> Puedo ver una imagen de los diferentes proyectos que el creador del portafolio ha construido (si no has construido ningún sitio web antes, usa marcadores de posición.)",
+        "<strong>Historia de usuario opcional:</strong> Puedo navegar a las diferentes secciones de la página web pulsando botones de navegación.",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/basic-ziplines.json]: La versión en inglés no tiene la palabra `opcional`.

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 3]
```diff
@@ -44,13 +44,13 @@
...
       "challengeType": 7,
-      "nameEs": "Translate",
+      "nameEs": "Únete a nuestro Subreddit",
       "descriptionEs": [
         [
           "http://i.imgur.com/DYjJuCG.gif",
-          "",
-          "",
-          ""
+          "Un gif que muestra como crear una cuenta de Reddit y unirse a Free Code Camp subreddit",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: `cómo` en lugar de `como`
`unirse al subreddit de Free Code Camp` en lugar de `unirse a Free Code Camp subreddit`

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 4]
```diff
@@ -44,13 +44,13 @@
...
+          "Nuestra comunidad tiene su propio subreddit en Reddit. Es una manera conveniente de hacer preguntas y compartir enlaces con nuestra comunidad entera. Si aún no dispones de una cuenta de Reddit, puedes crear una en pocos segundos - ni siquiera necesitas una dirección de correo electrónico. Entonces puedes pulsar el botón de \"subscribe\" para unirte a nuestro subreddit. También puede que te interesé suscribirte a otros subreddits listados en la columna lateral.",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: `interese` en lugar de `interesé`

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 5]
```diff
@@ -76,25 +76,25 @@
...
       "releasedOn": "February 10, 2016",      
-      "nameEs": "Translate",
+      "nameEs": "Lee noticias de Codificación en nuestro canal de publicaciones Medium",
       "descriptionEs": [
         [
           "http://i.imgur.com/FxSOL4a.gif",
-          "",
-          "",
-          ""
+          "Un gif mostrando como crear una cuenta en Medium.",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: `cómo` en lugar de `como`

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 6]
```diff
@@ -76,25 +76,25 @@
...
+          "Nuestra comunidad tiene un canal para publicaciones Medium, donde escribimos un montón de artículos sobre desarrollo de software. Si aún no tienes una cuenta Medium, puedes pulsar el enlace y registrarte con tu red social o ingresando tu dirección de correo electrónico (ellos enviarán un correo electrónico que debes abrirlo para crear tu cuenta.) Luego selecciona tus temas de interés y puedes continuar con los siguientes pasos.",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: `que debes abrir` en lugar de `que debes abrirlo`

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 7]
```diff
@@ -76,25 +76,25 @@
...
+          "https://www.medium.com"
         ],
         [
           "http://i.imgur.com/zhhywSX.gif",
-          "",
-          "",
-          ""
+          "Un gif mostrando como pulsar el botón \"follow\" para seguir las publicaciones de Free Code Camp.",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: `cómo` en lugar de `como`

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 8]
```diff
@@ -112,8 +112,8 @@
       "descriptionEs": [
         [
           "http://i.imgur.com/8rtyRY1.gif",
-          "",
-          "",
+          "Un gif mostrando como registrarse en Twitch.tv y seguir nuestro canal.",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: `cómo` en lugar de `como`

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 9]
```diff
@@ -112,8 +112,8 @@
...
+          "Nuestros campistas programan con frecuencia en vivo por Twitch.tv, un sitio web popular para streaming. Puedes crear una cuenta en menos de un minuto, luego sigue al canal de Free Code Camp. Cuando sigas al canal, verás la opción de noficación por correo electrónico cada vez que un campista esta en vivo. Puedes unirte a docenas de otros campistas y mirarlos programar, e interactuar en la sala de chat. Es divertido, una forma casual de aprender mientras miras gente construir proyectos.",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: `esté` en lugar de `esta`

**Comment by justinian336** [comment]
Thanks for the PR @cuent. I left a couple of comments, the rest seems ok. You can make these corrections along with those pointed out by @BerkeleyFalse and @bugron.

**Comment by BerkeleyFalse** [comment]
@cuent updated the pull request.

**Comment by BerkeleyFalse** [comment]
@cuent updated the pull request.

**Comment by bugron** [comment]
@cuent [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) you commits into one, please.

**Comment by cuent** [comment]
@BerkeleyFalse Sorry I used "closes" word like say the guidelines for Spanish translation. I'll keep in mind for the next time. @bugron and @justinian336  I hope everything is OK.

**Comment by BerkeleyFalse** [comment]
@cuent updated the pull request.

**Comment by vtamara** [comment]
@cuent please squash your commits, so when you go to the commits tab of this PR, you will see only one.

**Pull request closed without merging**